example-app/SantokuApp/src/features/demo-push-notification/services/usePushNotificationSendCommands.ts

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
+import {AppConfig} from 'bases/core/configs/AppConfig';
+import {useMutation} from 'react-query';
+
+import {Data} from '../types/Data';
+
+type PushNotificationParams = {
+  notificationTitle?: string;
+  notificationBody?: string;
+  data?: Data[];
+  badge?: string;
+  collapseId?: string;
+  contentAvailable?: boolean;
+  priority?: string;
+  interruptionLevel?: string;
+  relevanceScore?: string;
+  notificationCount?: string;
+  collapseKey?: string;
+  channelId?: string;
+};
+
+export const usePushNotificationSendCommands = () => {
+  const {mutateAsync: sendToMe, isLoading: isSendingToMe} = useMutation(
+    ({token, params}: {token: string; params: PushNotificationParams}) =>
+      axios.put(`${AppConfig.santokuAppBackendUrl}/api/sandbox/push-notification/single/${token}`, params),
+  );
+
+  const {mutateAsync: sendToAll, isLoading: isSendingToAll} = useMutation(
+    ({params}: {params: PushNotificationParams}) =>
+      axios.put(`${AppConfig.santokuAppBackendUrl}/api/sandbox/push-notification/all`, params),
+  );
```

Comment:
`useMutation` も関数を外に定義したほうが良いです？

```suggestion
const putToSingle = ({token, params}: {token: string; params: PushNotificationParams}) =>
  axios.put(`${AppConfig.santokuAppBackendUrl}/api/sandbox/push-notification/single/${token}`, params);

const putToAll = ({params}: {params: PushNotificationParams}) =>
  axios.put(`${AppConfig.santokuAppBackendUrl}/api/sandbox/push-notification/all`, params);

export const usePushNotificationSendCommands = () => {
  const {mutateAsync: sendToMe, isLoading: isSendingToMe} = useMutation(putToSingle);

  const {mutateAsync: sendToAll, isLoading: isSendingToAll} = useMutation(putToAll);
```